setup.py

```diff
@@ -1,4 +1,13 @@
 from setuptools import setup
+from distutils.command.install_scripts import install_scripts
+from shutil import move
+
+class strip_ext(install_scripts):
+    def run(self):
+        install_scripts.run(self)
+        for script in self.get_outputs():
```

Comment:
We inherit it as part of install_scripts, returns files from build_dir and install_dir.